huaweicloud/services/rds/resource_huaweicloud_rds_instance.go

```diff
@@ -592,22 +592,6 @@ func resourceRdsInstanceRead(ctx context.Context, d *schema.ResourceData, meta i
 		return diag.Errorf("error saving nodes to RDS instance (%s): %s", instanceID, err)
 	}
 
-	az1 := instance.Nodes[0].AvailabilityZone
```

Comment:
can we define the type of `availability_zone` to **schema.TypeSet**?

Reply:
> 

no, we can't, the first one of the availability_zone list is the master node.